src/DataManagement/Entries/Entry.cs

```diff
@@ -10,7 +11,9 @@ public class Entry : IEntry
     {
         public int Id { get; set; }
         public int UserId { get; set; }
+        [StringLength(25, ErrorMessage = "Too long title!")]
```

Comment:
It should limit Length to 25, and when longer entered this message should be displayed. But as far as I tested it just blocks entering chars when length is 25 in the field, so error message is redundant. Why I put this inside here? Just to show overall limitation, because same is used for each entry.